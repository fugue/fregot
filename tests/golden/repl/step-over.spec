{
    "command": "fregot",
    "arguments": ["repl", "step.rego"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${SPEC_NAME}.stderr"},
        {"stdout": "${SPEC_NAME}.stdout"}
    ],
    "stdin": [
        ":break step.test_step",
        "data.step.test_step",
        ":next",
        ":next"
    ]
}
