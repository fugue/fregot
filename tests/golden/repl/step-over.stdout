repl% repl% 14|   low_a
      [32m^^^^^[0m
repl(debug)% 15|   low_b
      [32m^^^^^[0m
repl(debug)% [32m(debug)[0m = [31mtrue[0m
[32m(debug)[0m finished
repl% 